client/html/src/Client/Html/Checkout/Standard/Process/Standard.php

```diff
@@ -282,8 +282,19 @@ public function process()
 
 			if( ( $form = $this->processPayment( $basket, $orderItem ) ) === null )
 			{
+				$args = array();
+				$config = array();
+
+				$services = $basket->getService( \Aimeos\MShop\Order\Item\Base\Service\Base::TYPE_PAYMENT );
+
+				if( ( $service = reset( $services ) ) !== false )
+				{
+					$args = array( 'code' => $service->getCode() );
+					$config = array( 'absoluteUri' => true, 'namespace' => false );
```

Comment:
The namespace is specific to TYPO3. Isn't that already disabled here?
https://github.com/aimeos/aimeos-typo3/blob/master/Configuration/TypoScript/setup.txt#L42

Reply:
It might well be, if this is the case then it's also the same for the exiting checks already present within the `processPayment` method?

Reply:
You are right, they can be removed as well